ua-snap/alaska-wildfires
# Integrate PurpleAir data layer

Adds PurpleAir sensor integration.

For testing, run a local instance of the data API on the `main` branch, and set the environment to point at that API endpoint: `export VUE_APP_SNAP_API_URL=http://localhost:5000` before running `npm run serve`.  

Testing:
 - Activate the Air quality sensor layer.
 - Click on a few popups to see how they look.
 - Hack it to get the different levels: at line 286 of `src/components/FireMap.vue`, add: `feature.properties.pm2_5_24hr = 55` to see things all set to Moderate, etc, for each category.  Make sure everything renders as expected.
 - Check the legend, look for any issues.
 - Note the introduction + check the links where PurpleAir / NASA are mentioned.  
 - Note the wee bit of code added to `MapLayer` so that we can have a tiny bit of documentation ("blurb") when the layer is activated.
